### ISSUE TITLE
docs: fix branch names

### ISSUE THREAD

**Issue by jkowalleck** [CycloneDX/cyclonedx-node-yarn#27]
before release, after branch "1.0-dev" was merged into "main", the current default-branch "1.0-dev" shall be set to "main".

this MUST also include updating all the docs, changing links from "1.0-dev" to "main" branch.

blocked by #6
